HACKING.adoc

```diff
@@ -43,6 +43,14 @@ make tests
 ----
 opam compiler-conf install
 ----
++
+With opam 2, create a local opam switch with the compiler installed from
+the current source directory:
++
+----
+opam switch create . --empty
+opam install .
+----
```

Comment:
Why use a local switch for this ?

Reply:
Why not? One benefit I see is that "get rid of all this stuff" operations such as `git clean -xfd` will also get rid of the switch.